src/Compilers/CSharp/Portable/Binder/ExpressionVariableFinder.cs

```diff
@@ -390,6 +393,16 @@ public override void VisitConstructorDeclaration(ConstructorDeclarationSyntax no
             }
         }
 
+        public override void VisitRecordDeclaration(RecordDeclarationSyntax node)
+        {
+            Debug.Assert(((RecordDeclarationSyntax)node).ParameterList is object);
+
+            if (node.BaseWithArguments is SimpleBaseTypeSyntax baseWithArguments)
+            {
+                VisitNodeToBind(baseWithArguments);
```

Comment:
>VisitNodeToBind [](start = 16, length = 15)

The scope rules implemented here are not supported by the specification at [https://github.com/dotnet/csharplang/blob/master/proposals/records.md](https://github.com/dotnet/csharplang/blob/master/proposals/records.md)

Reply:
> The scope rules implemented here are not supported by the specification at [https://github.com/dotnet/csharplang/blob/master/proposals/records.md](https://github.com/dotnet/csharplang/blob/master/proposals/records.md)

It is a well-known fact that the spec is behind. Please see [https://github.com/dotnet/csharplang/blob/master/meetings/2020/LDM-2020-06-01.md](https://github.com/dotnet/csharplang/blob/master/meetings/2020/LDM-2020-06-01.md)

---
In reply to: [436049242](https://github.com/dotnet/roslyn/pull/44876#discussion_r436049242) [](ancestors = 436049242)

Reply:
Please update the specification so this code can be checked against that.

---
In reply to: [436051230](https://github.com/dotnet/roslyn/pull/44876#discussion_r436051230) [](ancestors = 436051230,436049242)